samples/openthread/ncp/prj.conf

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+# ===============================
+# KERNEL OPTIONS
+# ===============================
+CONFIG_NEWLIB_LIBC=y
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_INIT_STACKS=y
+# ===============================
+
+# ===============================
+# LOGGING OPTIONS
+# ===============================
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_STATISTICS=y
+CONFIG_PRINTK=y
+CONFIG_OT_NCP_LOG_LEVEL_DBG=y
+CONFIG_NET_CONFIG_LOG_LEVEL_OFF=y
+
+CONFIG_LOG_BACKEND_SPINEL=y
+CONFIG_LOG_BACKEND_RTT=n
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_BOOT_BANNER=n
+CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE=2048
+CONFIG_LOG_BACKEND_SPINEL_LEVEL_INFO=y
+# ===============================
+
+# ===============================
+# NETWORKING OPTIONS
+# ===============================
+# ====================
+# OPENTHREAD OPTIONS
+# ====================
+# set OpenThread NCP architecture
+CONFIG_OPENTHREAD_NCP=y
+
+# enable OpenThread
+CONFIG_NET_L2_OPENTHREAD=y
+
+CONFIG_OPENTHREAD_DEBUG=y
+CONFIG_OPENTHREAD_L2_DEBUG=y
+CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF=y
+CONFIG_OPENTHREAD_CHANNEL=11
+# Use PANID 0xffff to be able to commission
+CONFIG_OPENTHREAD_PANID=65535
+
+# stack sizes configuration
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+CONFIG_NET_TX_STACK_SIZE=2048
```

Comment:
Possibly TX and RX stack size increase is no longer needed after zephyr upmerge (rx was reworked long time ago and I've reworked tx recently). test it and remove if so since you need to wait for upmerge.

Reply:
PR will not wait for Zephyr merge.